package.json

```diff
@@ -52,7 +52,6 @@
     "rand-token": "0.4.0",
     "sendgrid": "5.2.3",
     "should": "^13.2.3",
-    "swagger": "git://github.com/swagger-api/swagger-node.git#fffa399",
```

Comment:
In my opinion. Only this is required for this PR. Leave the other stuff for a different PR dedicated to removing the Swagger global dependency.